articles/service-fabric/service-fabric-backup-restore-service-trigger-restore.md

```diff
@@ -160,22 +168,31 @@ For _Named Partitioning_, the name value is compared to identify the target part
 
 #### Powershell using Microsoft.ServiceFabric.Powershell.Http Module
 
+Replace `account-name` with your storage account name.
```

Comment:
ambiguous. mention the parameter where the use needs to replace.

Reply:
updated